src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.test.tsx

```diff
@@ -81,3 +81,26 @@ it('Render a loading state checking search disabled', () => {
 
   expect(queryByPlaceholderText('Filter by name/url')).toHaveAttribute('disabled');
 });
+
+it('finds search box, enters text, and checks text occurrence', async () => {
+  (useRepositoryParams as jest.Mock).mockImplementation(() => ({ isLoading: false }));
+  (usePopularRepositoriesQuery as jest.Mock).mockImplementation(() => ({
+    isLoading: false,
+    data: { data: [defaultPopularRepository], meta: { count: 1, limit: 20, offset: 0 } },
+  }));
+
+  const { getByPlaceholderText, queryAllByText } = render(
+    <ReactQueryTestWrapper>
+      <PopularRepositoriesTable />
+    </ReactQueryTestWrapper>,
+  );
+
+  const searchBox = getByPlaceholderText('Filter by name/url');
+  fireEvent.change(searchBox, { target: { value: 'yourSearchText' } });
+
+  waitFor(() => {
+    const occurrences = queryAllByText('yourSearchText', { exact: false });
+    expect(occurrences).toBeTruthy();
```

Comment:
Read [this](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) on what "truthy means" (it just means not negative value) 
Query all by text returns an empty array "[]". 
<img width="424" alt="Screenshot 2024-01-24 at 11 50 04 AM" src="https://github.com/content-services/content-sources-frontend/assets/38083295/b03efa51-fd57-4a6d-9060-6f8175540c2f">

Reply:
It prints out an array in the boolean context and highlights if its true

Reply:
Reread the above